src/predict/api/helpers.py

```
@@ -400,22 +551,109 @@ def read_faulty_devices():
+                        {"title": tip["title"], "description": tip["description"]}
+                        for tip in static_tips.values()
+                    ]
+                
+                health_tips.append(tips_for_value)
+            
+            forecast["health_tips"] = health_tips
+    
     return results
+
+def add_forecast_health_tips_cached(
+    results: Dict, 
```

Ruff (0.8.2)

Undefined name `Dict`

(F821)